<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileViewWidget</class>
 <widget class="QWidget" name="FileViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_none"/>
     <widget class="TextEditorWidget" name="page_text"/>
     <widget class="ImageViewWidget" name="page_image"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewWidget</class>
   <extends>QWidget</extends>
   <header>ImageViewWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TextEditorWidget</class>
   <extends>QWidget</extends>
   <header>TextEditorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
